packages/queries/src/support/support.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  closeSupportTicket,
+  createReply,
+  createSupportTicket,
+  getTicket,
+  getTicketReplies,
+  getTickets,
+} from '@linode/api-v4/lib/support';
+import { createQueryKeys } from '@lukemorales/query-key-factory';
+import {
+  keepPreviousData,
+  useInfiniteQuery,
+  useMutation,
+  useQuery,
+  useQueryClient,
+} from '@tanstack/react-query';
+
+import type {
+  ReplyRequest,
+  SupportReply,
+  SupportTicket,
+  TicketRequest,
+} from '@linode/api-v4';
+import type {
+  APIError,
+  Filter,
+  Params,
+  ResourcePage,
+} from '@linode/api-v4/lib/types';
```

Comment:
Optional: clean up these imports

```suggestion
} from '@linode/api-v4';
```